libs/tokens/src/state/tokenLists/tokenListsStateAtom.ts

```diff
@@ -56,7 +56,7 @@ export const allListsSourcesAtom = atom((get) => {
 
 // Lists states
 export const listsStatesByChainAtom = atomWithStorage<TokenListsByChainState>(
-  'allTokenListsInfoAtom:v3',
+  'allTokenListsInfoAtom:v4',
```

Comment:
Migration? Otherwise users will lose previously manually added tokens lists

Reply:
No need to add a migration, this state just a cache.
This store is filled by `TokensListsUpdater`.
User-added lists are stored in `userAddedListsSourcesAtom`.